specification/elasticsan/resource-manager/Microsoft.ElasticSan/preview/2024-06-01-preview/elasticsan.json

```diff
@@ -1069,6 +1144,99 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ElasticSan/elasticSans/{elasticSanName}/deletedVolumes": {
+      "get": {
+        "tags": [
+          "Deleted Volumes List"
```

Comment:
Shouldn't it be soft-deleted?

Reply:
These are old changes, these aren't present in the latest file.